youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -591,6 +591,23 @@ def run(self, info):
         os.rename(encodeFilename(temp_filename), encodeFilename(filename))
         return info['__files_to_merge'], info
 
+    def can_merge(self):
+        # TODO: figure out merge-capable ffmpeg version
```

Comment:
Well, anyway we have to check some version agains the test set from #5661 to be sure.